### PR TITLE
lldpd: new, 1.0.17

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

lldpd: new, 1.0.17

https://github.com/lldpd/lldpd

Package(s) Affected
-------------------

- lldpd

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-network/lldpd/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=lldpd
+PKGDES="Implementation of IEEE 802.1ab (LLDP)"
```
Comment (MingcongBai) [inline app-network/lldpd/autobuild/defines]: Please be more descriptive, this is a toolkit right?

File: app-network/lldpd/autobuild/postinst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+getent group _lldpd &> /dev/null || \
```
Comment (MingcongBai) [inline app-network/lldpd/autobuild/postinst]: Please use systemd-sysusers instead.

File: app-network/lldpd/spec [hunk 3]
```diff
@@ -0,0 +1,3 @@
+VER=1.0.17
+SRCS="tbl::https://github.com/lldpd/lldpd/archive/refs/tags/$VER.tar.gz"
+CHKSUMS="sha256::8ea4115e061e0dceac59761b84d21e025c48d424e632d09d1201cfdf79309bff"
```
Comment (MingcongBai) [inline app-network/lldpd/spec:3]: Missing CHKUPDATE.

Comment (MingcongBai) [reply app-network/lldpd/spec:3]: Please use Anitya, release-monitoring.org

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-network/lldpd/spec (hunk 3 above)
Comment (jiegec) [reply app-network/lldpd/spec:3]: Done

File: app-network/lldpd/autobuild/postinst (hunk 2 above)
Comment (jiegec) [reply app-network/lldpd/autobuild/postinst]: Done

File: app-network/lldpd/autobuild/defines (hunk 1 above)
Comment (jiegec) [reply app-network/lldpd/autobuild/defines]: Changed to match https://lldpd.github.io/ headings

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**